frontend/pages/policies/PolicyPage/screens/RunQuery.tsx

```diff
@@ -93,7 +93,7 @@ const RunQuery = ({
       setupDistributedQuery(websocket);
       setCampaignState((prevCampaignState) => ({
         ...prevCampaignState,
-        campaign: returnedCampaign,
+        campaign: { ...prevCampaignState.campaign, returnedCampaign },
```

Comment:
I don't 100% follow this change -- is it to get default values into `campaign`?

Reply:
To make sure they _stay__ in `campaign`, as they'll already have been set from `DEFAULT_CAMPAIGN_STATE` and `returnedCampaign` [no longer includes as these 0 values](https://github.com/fleetdm/fleet/pull/27258/files/23807b10c42dd183f3dd35b7783477309c494e28#r2002066637)

In a follow-up I'll change the arg name here to `defaultCampaignState` which is a little clearer, since this is only called when opening the websocket